src/types/models/carSales.ts

```diff
@@ -1,5 +1,9 @@
 import { ApiSearchListing } from "../../types/models/listing"
 
+export interface CarSaleCount {
+  count: number
+}
+
```

Comment:
This is not needed anymore, now that we switched to number.

Reply:
True! thanks